packages/jest-reporters/src/default_reporter.ts

```diff
@@ -172,18 +172,18 @@ export default class DefaultReporter extends BaseReporter {
     result: TestResult,
   ) {
     this.log(getResultHeader(result, this._globalConfig, config));
-    const consoleBuffer = result.console;
-    if (consoleBuffer && consoleBuffer.length) {
+    if (result.console) {
       this.log(
         '  ' +
           TITLE_BULLET +
           'Console\n\n' +
           getConsoleOutput(
             config.cwd,
             !!this._globalConfig.verbose,
-            consoleBuffer,
+            result.console,
           ),
       );
+      result.console = undefined;
```

Comment:
I don't think the reporter should be responsible for mutating the testresult object. The memory consumption would be reintroduced with custom reporters. Can we move this to the runner or jest-core instead?

Reply:
Great feedback, moved to `jest-core`. To be honest, I just blindly followed the pattern of how `coverage` was previously released without thinking about it too carefully. Thanks for your review.